vtex/address-form
# CHK-2356: Replace babel with rollup in build script

#### What is the purpose of this pull request?

Replaces simple `babel` script with `rollup` in build process.

#### What problem is this solving?

Babel didn't transpile the newly added JSON files in the `react/country/data` directory, breaking the use of this library through NPM.

I replaced the build step to use Rollup, which is a fully-fledged build tool that we already use in several other repositories for its simplicity.

The configuration is quite simple, but it is powerful enough to cover every use case in this repository (and could also be used in other IO apps that publish to NPM).

#### How should this be manually tested?

You can run `yarn build` to see the build passing. I also did a thorough review of the generated `lib` directory using Babel and Rollup and the changes were minor (mostly internal variable names, and import/export).

#### Screenshots or example usage

N/A

#### Types of changes
